gnosis/safe/addresses.py

```diff
@@ -686,6 +686,30 @@
             "1.3.0",
         ),  # default singleton address
     ],
+    EthereumNetwork.OP_SEPOLIA_TESTNET: [
+        ("0x29fcB43b46531BcA003ddC8FCB67FFE91900C762", 7162879, "1.4.1+L2"),
```

Comment:
Please add first the addresses on https://github.com/safe-global/safe-deployments/tree/main for v1.4.1